src/toolboxconfig.ts

```diff
@@ -0,0 +1,163 @@
+// Configure your desired categories as objects in categories array
+// Configure category blocks as objects in blocks array
+export interface ToolboxCategoryBlocks {
+  id?: number,
+  tag: string,
+  paired: Boolean,
+  desc: String
+}
+
+export interface ToolboxCategory {
+  id: number,
+  name: String,
+  highlight: String,
+  blocks: Array<ToolboxCategoryBlocks>
```

Comment:
I suggest you to use the array short syntax `ToolboxCategoryBlocks[]` which is more common and less verbose.

Reply:
Awesome, that's indeed a lot clearer. Done for both array types in toolboxconfig.ts.